anabot-1.sh

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (C) 2021  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+TESTTYPE="ui smoke anabot skip-on-rhel-8"
+
+. ${KSTESTDIR}/functions.sh
+
+kernel_args() {
+    local tmp_dir="${1}"
+
+    # Provide an installation source.
+    local repo_url="$( echo ${KSTEST_URL} | sed -e 's/^--url=//' )"
+
+    # Provide the anabot recipe.
+    cp ${KSTESTDIR}/${name}.xml ${tmp_dir}/http/example.xml
+    local httpd_url="$(cat ${tmp_dir}/httpd_url)"
+    local recipe_url="${httpd_url}/example.xml"
+
+    echo "${DEFAULT_BOOTOPTS} inst.graphical  anabot=${recipe_url} inst.repo=${repo_url}"
+}
+
+prepare_updates() {
+    local tmp_dir="${1}"
+    local updates_dir="${tmp_dir}/updates"
+    local updates_img="${tmp_dir}/updates.img"
+
+    local anabot_branch="main"
+    local anabot_url="https://pagure.io/anabot.git"
+    local anabot_img="${tmp_dir}/anabot.tar.gz"
+
+    # Use a fork.
+    # FIXME: Don't use the fork.
+    local anabot_url="https://pagure.io/forks/vponcova/anabot.git"
```

Comment:
A typo, "forks" -> "fork" ?

Reply:
I have checked the GIT URL and it is really `forks`.

Reply:
Ah, sorry for the noise, you are right.